wp-includes/widgets/class-wp-widget-visual-text.php

```diff
@@ -149,18 +154,31 @@ public function form( $instance ) {
 				'text' => '',
 			)
 		);
+		?>
+		<input id="<?php echo $this->get_field_id( 'title' ); ?>" name="<?php echo $this->get_field_name( 'title' ); ?>" class="title" type="hidden" value="<?php echo esc_attr( $instance['title'] ); ?>">
+		<input id="<?php echo $this->get_field_id( 'text' ); ?>" name="<?php echo $this->get_field_name( 'text' ); ?>" class="text" type="hidden" value="<?php echo esc_attr( $instance['text'] ); ?>">
+		<?php
+	}
 
-		$title = sanitize_text_field( $instance['title'] );
```

Comment:
This was redundant so I removed it. The title is sanitized in the `update` method. All we need here is outputting the value with the appropriate escaping.